intel_extension_for_transformers/llm/runtime/graph/CMakeLists.txt

```diff
@@ -91,7 +91,7 @@ option(NE_GELU_VEC               "neural_engine: enable vec in gelu"
 if (NE_GELU_VEC)
     add_compile_definitions(NE_GELU_USE_VEC)
 endif()
-option(PYTHON_API              "neural_engine: use python api"                                  OFF)
+option(NE_PYTHON_API              "neural_engine: use python api"                                  OFF)
```

Comment:
should we use LLM_RUNTIME_PYTHON_API?

Reply:
NE_ PYTHON_API is named after other environment variables, and the prefix is a legacy. If changes are made, it will involve many files